lang/python/python3/Makefile

```diff
@@ -269,18 +280,21 @@ define Py3Package/python3-light/filespec
 $(subst $(space),$(newline),$(foreach lib_file,$(PYTHON3_LIB_FILES_DEL),-|$(lib_file)))
 endef
 
-define Py3Package/python3-base/install
+define Package/libpython3/install
 	# Adding the lib-dynload folder (even just empty) suppresses 2 warnings when starting Python
 	$(INSTALL_DIR) $(1)/usr/lib/python$(PYTHON3_VERSION)/lib-dynload/
-	$(INSTALL_DIR) $(1)/usr/bin
-	$(LN) python$(PYTHON3_VERSION) $(1)/usr/bin/python3
-	$(LN) python$(PYTHON3_VERSION) $(1)/usr/bin/python
 	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON3_VERSION).so* $(1)/usr/lib/
   # This depends on being called before filespec is processed
 	$(SED) 's|$(TARGET_AR)|ar|g;s|$(TARGET_CROSS)readelf|readelf|g;s|$(TARGET_CC)|gcc|g;s|$(TARGET_CXX)|g++|g' \
 		$(PKG_INSTALL_DIR)/usr/lib/python$(PYTHON3_VERSION)/_sysconfigdata.py
```

Comment:
I think these lines modifying _sysconfigdata.py can be kept/moved into `Py3Package/python3-base/install` since the file is installed by python3-base.

Reply:
i'm feeling `_sysconfigdata.py` may be a little better suited under `libpython`
but i won't object much to moving it;
we can move it;

initially i was thinking of cases where some users may want to install just `libpython` for some weird reasons;
that would be a sort of "python-devel" [or libpython-dev] let's say;
getting to "python-devel" is a long way and may require more elements; so whether `_sysconfigdata.py` stays in python3-base or libpython3, doesn't make much difference;

Reply:
_sysconfigdata.py was always in python3-base (it is included in `PYTHON3_BASE_LIB_FILES`). This sed line only replaced references to `$(TARGET_CC)`/`$(TARGET_CXX)`/etc with `gcc`/`g++`/...